storage_service/locations/api/resources.py

```diff
@@ -272,6 +276,26 @@ def get_objects(self, space, path):
         message = _('This method should be accessed via a versioned subclass')
         raise NotImplementedError(message)
 
+    def default(self, request, **kwargs):
```

Comment:
Is there a reason that you can not using the `_custom_endpoint` decorator? If not, then maybe you should for module-internal consistency. If yes, could you add a brief comment indicating why?